pay-api/src/pay_api/services/base_payment_system.py

```diff
@@ -238,8 +237,7 @@ def _publish_refund_to_mailer(invoice: InvoiceModel):
                 'filingDescription': filing_description
             })
         current_app.logger.debug(f'Publishing payment refund request to mailer for {invoice.id} : {q_payload}')
-        publish_response(payload=q_payload, client_name=current_app.config.get('NATS_MAILER_CLIENT_NAME'),
-                         subject=current_app.config.get('NATS_MAILER_SUBJECT'))
+        gcp_queue_publisher.publish_to_queue(q_payload)
```

Comment:
Needed to use `gcp_queue_publisher.publish_to_queue` instead of `publish_to_queue` mocks were having a hard time picking up the latter